service/http/topic.go

```diff
@@ -67,6 +67,52 @@ type topicEventJSON struct {
 	PubsubName string `json:"pubsubname"`
 }
 
+func (in topicEventJSON) GetData() (data any, rawData []byte) {
```

Comment:
Why this change, where I split this out in a separate function, you may ask?

Well, ask the linter and its complaining about "cyclomatic complexity"...

Reply:
Can it be a private method instead?

Reply:
Sure, done